implementation/src/main/java/io/smallrye/mutiny/groups/MultiOnSubscribe.java

```diff
@@ -22,9 +22,9 @@
  *
```

Comment:
should we rename this class to MultiOnSubscription?

Reply:
That'd break runtime compatibility.